nlp_apps.ipynb

```diff
@@ -522,7 +520,37 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": 39,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "#removing stopwords\n",
+    "from nltk.corpus import stopwords\n",
```

Comment:
We want to try to minimize the use of third-party libraries. The point of the nlp module is to have basic implementations of standard functions used in the domain. Importing from nltk is the opposite of what we want to do.

Reply:
Thanks!